app/controllers/assessments_controller.rb

```diff
@@ -3,7 +3,7 @@ class AssessmentsController < ApplicationController
 
   # GET /assessments
   def index
-    @assessments = Assessment.all
+    @assessments = Assessment.all || []
```

Comment:
More specifically, `|| []` returns the value after the `||` in case the value before it is falsey. 
`3 || [] => 3`
`nil || [] => []`
`false || 2 => 2`

You're right that it's unnecessary here given that `.all` will never return nil.